docs/UserGuide.adoc

```diff
@@ -326,7 +327,7 @@ You can reopen the certain issue on github when you want to work on it. +
 
 * `reopenissue 1`
 * `reopenissue 3`
-
+//end::issues[]
```

Comment:
you may leave a new line between the tag and the "task list"

Reply:
Done 👍